### DISCUSSION TITLE
🐞 webhook to discord

### DISCUSSION THREAD

**Discussion by aldiwahyudin** [aldinokemal/go-whatsapp-web-multidevice#222] [Announcements]
### What happened?

I think maybe for webhook settings it can be implemented in a separate file because there are some differences in JSON format. 

### Reproduction Steps

```bash
//discord webhook standar
    dcBody := map[string]interface{}{
    		"content":        message.Text,
    		"username":       evt.Info.PushName,
    }
    dcpostBody, err := json.Marshal(dcBody)
    logrus.Info("webhook discord :", string(dcpostBody))
    resp, err := http.Post(config.WhatsappWebhook, "application/json", bytes.NewBuffer(dcpostBody))
        if err != nil {
                fmt.Println("Error sending request:", err)
        } else {
                defer resp.Body.Close()
                fmt.Println("Response status:", resp.Status)
        }
    //end diccord tool for web hook
```

### Application Version

4.18

### Verification

- [X] I have checked for existing closed issues and discussions.

**Comment by aldinokemal** [comment]
I don't think this app has to follow Discord or any app rules. Right now, you just need to deal with this problem by making your own middleware. Or you can tell me more about your issue because I'm not very familiar with Discord
